_posts/en/newsletters/2023-12-06-newsletter.md

```diff
@@ -0,0 +1,256 @@
+      one has two incomparable chunkings for a graph, then another
+      chunking must exist which is strictly better than both".  That
+      means an effective method for merging two different incomparable
+      chunkings can be a powerful tool for improving miner
+      profitability.  For example, Alice and Bob could both try
+      to find the optimal chunking and could share their results with
+      each other in a way that would guarantee they'd both obtain a
+      third chunking that was at least as good as either of their
+      original chunkings---and possibly better.  Alice and Bob could be
+      any node operators, although they would gain the most benefit if
+      they were miners.
+
+    - [Post-cluster package RBF][cluster rbf] discusses an alternative
+      to the rules currently used for [replace by fee][topic rbf], which
+      are known to cause multiple problems, including [pinning
+      attacks][topic transaction pinning].  When a valid replacement of
+      one or more transactions is received, a temporary version of all
+      the clusters it affects can be made and their updated chunking
+      derived.  This can be compared to the chunking of the original
+      clusters that are currently in the mempool (that don't include the
+      replacement).  If the chunking with the replacement always earns equal or
+      more fees than the original for any number of vbytes, and if it
+      increases the total amount of fees in the mempool by enough to pay
+      for its relay fees, then it should be included in the mempool.
+
+        This evidence-based evaluation can replace several
+        [heuristics][mempool replacements] currently used in Bitcoin
+        Core to determine whether a transaction should be replaced,
+        potentially improving the RBF rules in several areas, including
+        proposed [package relay][topic package relay] for replacements.
+        Several [other][cluster rbf-old1] threads [also][cluster
+        rbf-old2] discussed [this][cluster rbf-old3] topic.
+
+- **Testing with warnet:** Matthew Zipkin [posted][zipkin warnet] to
+  Delving Bitcoin with the results of some simulations he's run using
+  [warnet][], a program that launches a large number of Bitcoin nodes
+  with a defined set of connections between them (usually on a test
+  network).  Zipkin results show the extra memory that would be used if
+  several proposed changes to peer management code are merged (either
+  independently or together).  He also notes that he's excited to use
+  simulations for testing other proposed changes and to quantify the
+  effect of proposed attacks.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+Before each [major Bitcoin Core release][], extensive testing by the
```

Comment:
I think the summary for this PR Review Club is great and we could likely drop all of the Q&A section below, which I thought added less value.

Reply:
> I think the summary for this PR Review Club is great and we could likely drop all of the Q&A section below, which I thought added less value.

Thanks, and I'm perfectly okay with that suggestion. In fact, I almost did that, but thought that _maybe_ keeping it would give the reader a feel for the level and kinds of detail and depth that occur in these release testing meetings -- I agree that the actual content is pretty low-information. But OTOH the reader can simply click the link to view the actual discussion log, so maybe that's not a good reason.

I just noticed, that there is no actual link to the review club page, I'll fix that right now (with a non-force push, new commit).

Feel free to remove the questions, or I can do that, or we can wait to see what others think. But I do like your suggestion.

Reply:
I'm +0.1 on keeping the QA.  I'm happy with whatever Larry and Mike decide.